java/server/src/org/openqa/selenium/grid/data/BrowserInfo.java

```diff
@@ -0,0 +1,59 @@
+// Licensed to the Software Freedom Conservancy (SFC) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The SFC licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.openqa.selenium.grid.data;
+
+import java.util.Map;
+import java.util.HashMap;
+
+import static java.util.Collections.unmodifiableMap;
+
+public class BrowserInfo {
+  private String browser;
+  private Integer count;
```

Comment:
If it's never valid for this to be `null` (and I'd suggest it's not 😄 ) then this should be an `int`